webofneeds/won-owner/src/main/resources/db/migration/V1__initial_database.sql

```diff
@@ -0,0 +1,164 @@
+BEGIN;
+CREATE TABLE bastate
+(
```

Comment:
The live owner database has been created newly since this pull request has been created, i am not sure if we should also recreate this ddl then?